libcudacxx/test/libcudacxx/cuda/for_each_canceled/for_each_canceled.pass.cpp

```diff
@@ -0,0 +1,220 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of libcu++, the C++ Standard Library for your entire system,
+// under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: pre-sm-100
```

Comment:
Given we have a fallback for before SM100, why can't we test on those architectures?

Reply:
```suggestion
```

We can now.